README.yaml

```diff
@@ -154,6 +154,8 @@ usage: |2-
         non_key_attributes = ["HashKey", "RangeKey"]
       }
     ]
+
+    replicas = ["us-east-1"]
```

Comment:
Tried running `make readme` but got this:

```
❯ make readme
* Package gomplate already installed
* Package terraform-docs already installed
make: gomplate: No such file or directory
make: *** [readme/build] Error 1
```

Reply:
I guess you forgot to run `make init`. btw you can open PR from your account (not organisation) to make it possible for our bot to generate `README`